### DISCUSSION TITLE
세번째 모임 후기 스레드

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/growth-collector#15] [모임 후기!]
이제 목표 달성을 위해 구체적인 액션 아이템을 설계하고, 진행에 어떤 어려움이 있는지 공유해볼 수 있었던 시간인 것 같습니다!
계속해서 성장을 수집하고, 서로의 목표를 향해 성장해나가는 우리들이 되길 바래봅니다!

참가자: 찬란, 창엽, 영승, 윤희
새로운 참가자: 의형

- 2주간 진행할 액션 아이템을 구체화하고, 진행하기!
- AI 대가, 전문가 리스트업! / 해외 10명, 국내 10명 이상 정도..? ([저기!](https://github.com/Pseudo-Lab/growth-collector/blob/main/list.md))

ps. 사진을 또 못찍어서 너무 아쉬워요! 다음엔 진짜... 꼭... 찍고 시작하기로...!

**Comment by yunhee1** [comment]
좋은 PM의 요건이 무엇인지에 대해 의견을 들을 수 있어서 좋았습니다. 말씀해주신 내용을 바탕으로 제가 발전시켜나갈 방향을 잡을 수 있었습니다!
또한 이야기 나누었던 5년 후 내 모습, 나의 성장을 방해하는 것들을 생각해보고 이야기해볼 수 있는 기회를 가져서 너무 좋았습니다 :) !

사진을 깜빡했네요 정말,, 다음에 만나면 꼭 찍어요!!!📷
